submissions/israelOladosu/database/init.sql

```diff
@@ -0,0 +1,120 @@
+CREATE DATABASE farmvile;
+
+use farmvile;
+
+-- categories table 
+CREATE TABLE categories (
+    id INT NOT NULL AUTO_INCREMENT,
+    category_name VARCHAR(50) NOT NULL,
+    PRIMARY KEY(id)
+);
+
+-- admins table
+CREATE TABLE admins (
+    id INT NOT NULL AUTO_INCREMENT,
+    firstname VARCHAR(50) NOT NULL,
+    lastname VARCHAR(50) NOT NULL,
+    email VARCHAR(100) NOT NULL,
+    admin_password VARCHAR(100) NOT NULL,
+    phone_number VARCHAR(21) NOT NULL,
+    PRIMARY KEY(id)  
+);
+
+-- products table
+CREATE TABLE products (
+    id INT NOT NULL AUTO_INCREMENT,
+    product_name VARCHAR(50) NOT NULL,
+    product_description VARCHAR(150) NOT NULL,
+    product_image BLOB,
+    unit_price INT NOT NULL,
+    stock_level INT NOT NULL,
+    product_status VARCHAR(20) NOT NULL,
+    admin_id INT NOT NULL,
+    category_id INT NOT NULL,
+    PRIMARY KEY(id),
+    FOREIGN KEY(admin_id) REFERENCES admins(id) ON DELETE CASCADE,
+    FOREIGN KEY(category_id) REFERENCES categories(id) ON DELETE CASCADE
+);
+
+-- registered customers table
+CREATE TABLE customers (
+    id INT NOT NULL AUTO_INCREMENT,
+    firstname VARCHAR(50),
+    lastname VARCHAR(50),
+    email VARCHAR(100) NOT NULL,
+    customer_password VARCHAR(100) NOT NULL,
+    primary KEY(id)
+);
+
+-- registered customerd addresses
+CREATE TABLE addresses (
+    id INT NOT NULL AUTO_INCREMENT,
+    street_address VARCHAR(250) NOT NULL,
+    city VARCHAR(50) NOT NULL,
+    state_name VARCHAR(20) NOT NULL,
+    zipcode INT,
+    country VARCHAR(25) NOT NULL,
+    phone_number VARCHAR(21) NOT NULL,
+    customer_id INT,
+    PRIMARY KEY(id),
+    FOREIGN KEY(customer_id) REFERENCES customers(id) ON DELETE CASCADE
+);
+
+-- orders table
+CREATE TABLE orders (
+    id INT NOT NULL AUTO_INCREMENT,
+    order_amount INT NOT NULL,
```

Comment:
well done, please use `DECIMAL` for amount. Find out how it is used

Reply:
Done... Thank you